.github/workflows/linux.yml

```
@@ -749,10 +749,12 @@ jobs:
     defaults:
       run:
         shell: bash
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest-4-cores
     env:
       INSTALL_DIR: ${{ github.workspace }}/install
       INSTALL_TEST_DIR: ${{ github.workspace }}/install/tests
+      PARALLEL_TEST_SCRIPT: ${{ github.workspace }}/install/tests/functional_test_utils/run_parallel.py
```

```suggestion
      PARALLEL_TEST_SCRIPT: $INSTALL_TEST_DIR/functional_test_utils/run_parallel.py
```
can we somehow reuse `INSTALL_TEST_DIR`?